float32[2] neck_q
